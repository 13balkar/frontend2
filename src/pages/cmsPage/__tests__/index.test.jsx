import * as React from 'react';
import { render } from '@testing-library/react';
import Cms from '../';
import Navbar from '../../../components/navbar';
jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Cms', () => {
  it('should render the cms', () => {
    const { asFragment } = render(<Cms />);
    expect(asFragment()).toMatchSnapshot();
  });
});
