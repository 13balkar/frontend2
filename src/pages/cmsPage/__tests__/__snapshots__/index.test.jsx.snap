// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cms should render the cms 1`] = `
<DocumentFragment>
  <div
    class="cms"
  >
    <div
      class="navbar"
    >
      <div
        class="heading"
      >
        <h1>
          CMS+
        </h1>
      </div>
      <div
        class="list"
      >
        <p>
          COLLECTION TYPES
        </p>
      </div>
      <div
        class="builder"
      >
        <p>
          CONTENT TYPE BUILDER
        </p>
      </div>
    </div>
    <div
      class="contents"
    >
      <div
        class="content-heading"
      >
        <h1>
          Content Types
        </h1>
      </div>
      <div
        class="content-body"
      >
        <div
          class="content-list"
        />
        <div
          class="content-builder"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Cms should render the cms with a view 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <div
      class="heading"
    >
      <h1>
        CMS+
      </h1>
    </div>
    <div
      class="list"
    >
      <p>
        COLLECTION TYPES
      </p>
    </div>
    <div
      class="builder"
    >
      <p>
        CONTENT TYPE BUILDER
      </p>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Homepage should render the homepage 1`] = `
<DocumentFragment>
  <div
    class="cms"
  >
    <div
      class="navbar"
    >
      <div
        class="heading"
      >
        <h1>
          CMS+
        </h1>
      </div>
      <div
        class="list"
      >
        <p>
          COLLECTION TYPES
        </p>
      </div>
      <div
        class="builder"
      >
        <p>
          CONTENT TYPE BUILDER
        </p>
      </div>
    </div>
    <div
      class="contents"
    >
      <div
        class="content-heading"
      >
        <h1>
          Content Types
        </h1>
      </div>
      <div
        class="content-body"
      >
        <div
          class="content-list"
        />
        <div
          class="content-builder"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
